packages/dds/merge-tree/src/properties.ts

```diff
@@ -21,10 +21,6 @@ export interface IConsensusValue {
 	value: any;
 }
 
-/**
- * @deprecated - This functionality was not intended for public export and will
- * be removed in a future release.
- */
 export function combine(
```

Comment:
@connorskees While integrating #17908, I noticed that combine, extend, etc. are no longer marked as deprecated but are still exported. Is that intentional?

Reply:
Oh, my mistake. They aren't exported from the package, only the file. Ignore.